packages/sync/src/DataSyncClientOptions.ts

```diff
@@ -0,0 +1,30 @@
+import { PersistedData, PersistentStore } from "./PersistentStore";
+
+declare var window: any;
+
+/**
+ * Contains all options required to initialize SDK
+ * @see DefaultOptions for defaults
+ */
+export interface DataSyncClientOptions {
+  /**
+   * Describes name of the field used as ID
+   */
+  defaultObjectIdField?: string;
+
+  /**
+   * Storage solution
+   */
+  storage?: PersistentStore<PersistedData>;
```

Comment:
Love it!

Reply:
If you like this like this PR upstream: https://github.com/helfer/apollo-link-queue
Hopefully we could get this merged and get official support for persistence.